fluent-bit-kinesis.go

```diff
@@ -105,6 +107,10 @@ func newKinesisOutput(ctx unsafe.Pointer, pluginID int) (*kinesis.OutputPlugin,
 		appendNL = true
 	}
 
+	if replaceDots == "" {
+		logrus.Infof("[kinesis %d] No replacement provided. Do not need to replace dots with other symbols.", pluginID)
+	}
```

Comment:
You already print the value of this config option in the plugin initialization, remove this info statement, its not needed

Reply:
I see. Will remove this statement